crates/sovereign-sdk/full-node/db/sov-db/src/schema/tables.rs

```diff
@@ -258,6 +259,11 @@ define_table_with_default_codec!(
     (L2Witness) BatchNumber => Vec<u8>
 );
 
+define_table_with_default_codec!(
+    /// The primary source of witness by L2 height
+    (L2StateRoot) BatchNumber => Vec<u8>
```

Comment:
Can witness be of an arbitrary size?

Reply:
This will be handled with https://github.com/chainwayxyz/citrea/issues/880